docs/source/user_guide/automl.ipynb

```diff
@@ -607,10 +607,12 @@
    "outputs": [],
    "source": [
     "X, y = evalml.demos.load_breast_cancer()\n",
+    "X, y = X[:250], y[:250]\n",
```

Comment:
Is this necessary? You already set `n_rows` at the top

Reply:
This is a different dataset from fraud! Let me time if I can use the full breast cancer data.

Reply:
Yea I think shrinking the model families and doing one less batch are sufficient so I'll use the full dataset!